docs/src/_cloud/fees.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Cloud Fees"
+hidden: true
```

Comment:
This doesn't seem to exclude it from the ToC nor search. Is that expected?

<img width="203" alt="Screenshot 2023-03-31 at 14 46 04" src="https://user-images.githubusercontent.com/16805946/229226783-163bf6e2-8581-4502-8cdb-342c95ee3505.png">
<img width="323" alt="Screenshot 2023-03-31 at 14 45 48" src="https://user-images.githubusercontent.com/16805946/229226785-d9801c14-9105-41fe-bcc9-7d8c4eb3a806.png">

Reply:
@edgarrmondragon Good catch, I just copied what we had for cloud/sla etc. It's not a huge problem if it shows up in those places, as long as it's not in the sidebar.

Reply:
@DouweM cool. Also, [it looks like this](https://deploy-preview-7469--meltano.netlify.app/cloud/fees)
<img width="448" alt="Screenshot 2023-03-31 at 14 53 52" src="https://user-images.githubusercontent.com/16805946/229228142-c8665c53-e65d-4cbf-ab77-83ca89c5b2b2.png">

If that's not intended, I think `layout: doc` needs to be added to the frontmatter.